### DISCUSSION TITLE
Have 'Go Back' option during install steps

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2761] [Ideas]
During installation steps it would be really helpful to have a 'Go Back' option to back up to previous screens in case of mistyping or something.  Currently there's only a 'Cancel' option, which ends the process and requires the install to be completely restarted.  This is cumbersome if trying to work through errors.

**Comment by weslambert** [comment]
Unfortunately, TMK, this is a limitation of Whiptail, and we aren't able to easily accommodate this at this time.  We certainly understand how this can be frustrating, and this may be addressed if/when we move to another menu system.
